src/app/codelabs/react/introduction/exercise/test.js

```diff
@@ -0,0 +1,79 @@
+import {
+  babylon,
+  babel_traverse as traverse,
+  babel_types as T
+} from '../code';
+
+export default function test(sourceCode) {
+  const tests = [
+    {
+      instruction: `Create a function named App`,
+      condition: ({ node }) =>
+        (T.isFunctionDeclaration(node) && node.id.name === 'App') ||
+        (T.isVariableDeclarator(node) && node.id.name === 'App' && T.isFunctionExpression(node.init)) ||
+        (T.isVariableDeclarator(node) && node.id.name === 'App' && T.ArrowFunctionExpression(node.init))
```

Comment:
3x node.id.name === 'App'
2x T.isVariableDeclarator(node)

Can you clean this up?